src/components/ViewList.js

```diff
@@ -35,6 +39,16 @@ let ViewList = ({
       </select>
       <button onClick={() => resetFilter()}>Reset</button>
     </label>
+    <label>
+      Filter Performance:
+      <select value={filter} onChange={event => changeFilter(event)}>
+        <option value="All">All</option>
+        <option value="Researching">Strong</option>
+        <option value="Pending">Good</option>
+        <option value="Approved">Weak</option>
+      </select>
+      <button onClick={() => resetFilter()}>Reset</button>
+    </label>
```

Comment:
IN DEVELOPMENT: Add additional filters on other columns to further narrow down results. (Table population does not yet use this filter.)